integration-core/src/main/java/org/symphonyoss/integration/core/bootstrap/IntegrationBootstrapContext.java

```diff
@@ -19,21 +19,29 @@
 import static org.symphonyoss.integration.logging.DistributedTracingUtils.TRACE_ID;
 
 import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.core.LoggerContext;
+import org.apache.logging.log4j.core.config.Configuration;
+import org.apache.logging.log4j.core.config.LoggerConfig;
```

Comment:
These imports are not necessaries. You also shouldn't use Log4j classes directly. Rather than use SL4J classes.